src/model.rs

```diff
@@ -1299,7 +1298,7 @@ mod tests {
                 .unwrap();
             collections.try_merge(collections_to_merge).unwrap();
             let bus_mode = collections.physical_modes.get(BUS_PHYSICAL_MODE).unwrap();
-            assert_eq!(42f32, bus_mode.co2_emission.unwrap());
+            assert!((bus_mode.co2_emission.unwrap() - 42f32).abs() < std::f32::EPSILON);
```

Comment:
Can't you use `relactive_eq!` like object.rs here ?
(same for all `EPSILON` occurences I guess)

Reply:
yes, I did that before using `approx`  for the test I was blocked on but I didn't have the courage to change that

Reply:
done